test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -510,17 +539,17 @@ bool EVMInstructionInterpreter::accessMemory(u256 const& _offset, u256 const& _s
 {
 	if (_size == 0)
 		return true;
-	else if (((_offset + _size) >= _offset) && ((_offset + _size + 0x1f) >= (_offset + _size)))
```

Comment:
Is there really enough value in reordering the comparison here to complicate the review due to it?

Reply:
The original order is quite unnatural to read and it actually took me a moment to realize it's just "a <= b <= c" check when I needed to read this code. I thought it was worth a small tweak for anyone who may encounter it later. It did not seem worth creating a separate PR for it though.

Reply:
Moved to #14790.